src/extension_utils/utils.ts

```diff
@@ -607,47 +592,38 @@ export const setupEnv = async (
                             installChoice: vscode.MessageItem | undefined
                         ) => {
                             if (installChoice === DialogResponses.INSTALL_NOW) {
-                                if (
-                                    !(await installDependencies(
-                                        context,
-                                        pythonExecutableName
-                                    ))
-                                ) {
-                                    vscode.window
-                                        .showErrorMessage(
-                                            CONSTANTS.ERROR
-                                                .DEPENDENCY_DOWNLOAD_ERROR,
-                                            DialogResponses.READ_INSTALL_MD
-                                        )
-                                        .then(
-                                            (
-                                                selection:
-                                                    | vscode.MessageItem
-                                                    | undefined
-                                            ) => {
-                                                if (
-                                                    selection ===
-                                                    DialogResponses.READ_INSTALL_MD
-                                                ) {
-                                                    open(
-                                                        CONSTANTS.LINKS.INSTALL
-                                                    );
-                                                }
+                                await installDependenciesWrapper(
+                                    context,
+                                    pythonExecutableName
+                                );
+                            } else {
+                                await vscode.window
+                                    .showInformationMessage(
+                                        CONSTANTS.INFO.ARE_YOU_SURE,
+                                        DialogResponses.INSTALL_NOW,
+                                        DialogResponses.DONT_INSTALL
+                                    )
```

Comment:
I think it would be ok to remove this prompt, as it is the third prompt in a row asking about installing dependencies. Up to you.

Reply:
perhaps we should leave it in for now, since people who just close popups on instinct should be advised that the extension won't work without installing dependencies! Maybe we can incorporate this info into an earlier popup later on :0